docs/masking-providers.md

```diff
@@ -89,13 +89,21 @@ options and their default values.
 >   The interval's width is configurable, but is constant across the domain of values.  The range
 >   is inclusive on the lower bound and exclusive on the upper bound. The format in which the interval is presented is 
 >   configurable.  For examples of valid formats, see the java documentation for java.util.Formatter.
-
-| **Option name**       | **Type** | **Description**                         | **Default value** |
-|-----------------------|----------|-----------------------------------------|-------------------|
-| binSize               | Integer  | The range of the interval               | 5                 |
-| format                | String   | The format of the replacement range     | %s-%s             |
-| startValue            | Integer  | The lower bound of the range            | 0                 |
-| useStartValue         | Boolean  | Use the startValue as the lower bound   | false             |
+>   For threshold values, it uses a constant values. For example if values is 3, its under threshold value would be >10.
+>   Useful for age generalizations: <8, 5-10, 10-15, ..., 85-90, 90+.
+
+| **Option name**                      | **Type** | **Description**                         | **Default value** |
+|--------------------------------------|----------|-----------------------------------------|-------------------|
+| binSize                              | Integer  | The range of the interval               | 5                 |
+| format                               | String   | The format of the replacement range     | %s-%s             |
+| startValue                           | Integer  | The lower bound of the range            | 0                 |
+| useStartValue                        | Boolean  | Use the startValue as the lower bound   | false             |
+| useSingleBucketOverThreshold         | Boolean  | Use the startValue as the lower bound   | false             |
+| singleBucketOverThresholdValue       | Double   | Use the startValue as the lower bound   | 90.0              |
```

Comment:
When updating the description, be sure to mention the value must be greater than or equal to this given value to add it to the special interval.